src/frontend/src/utils/userNumber.test.ts

```diff
@@ -90,11 +91,12 @@ function testLocalStorage(
 ) {
   test(name, () => {
     localStorage.clear();
-    if (opts?.before !== undefined) {
-      setLocalStorage(opts.before);
+    const before = opts?.before;
+    if (nonNullish(before)) {
+      setLocalStorage(before);
     }
     fn();
-    if (opts?.after !== undefined) {
+    if (nonNullish(opts) && nonNullish(opts?.after)) {
```

Comment:
would it make sense to do the same for `after` as we do with `before`?

Reply:
sure go for it